constraint/pkg/apis/templates/v1alpha1/mutationtemplate_types_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+	"golang.org/x/net/context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestStorageMutationTemplate(t *testing.T) {
+	key := types.NamespacedName{
+		Name:      "foo",
+		Namespace: "default",
+	}
+	created := &MutationTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "foo",
+			Namespace: "",
+		}}
+	g := gomega.NewGomegaWithT(t)
+
+	// Test Create
+	fetched := &MutationTemplate{}
+	g.Expect(c.Create(context.TODO(), created)).NotTo(gomega.HaveOccurred())
```

Comment:
I see that the existing tests use context.TODO(), but a single context.Background() instantiated at the start of this test would be consistent with what's recommended by the golang docs:
https://golang.org/pkg/context/#Background

Reply:
I believe this is autogenerated code from kubebuilder

Reply:
+1 to "it's autogenned"